official/transformer/v2/transformer.py

```diff
@@ -103,6 +103,9 @@ def call(self, x, training):
           outputs: [batch_size, decoded length]
           scores: [batch_size, float]}
     """
+    if not tf.distribute.in_cross_replica_context() and len(x) == 1:
```

Comment:
Leave a comment and bug link if relevant as to why this is needed, it seems odd that we have to do something special.

Reply:
Figured out why this was needed and changed the mode such that this is no longer required.